src/Raven.Client/Documents/Commands/Batches/PutCommandDataBase.cs

```diff
@@ -8,8 +8,8 @@ namespace Raven.Client.Documents.Commands.Batches
 {
     internal class PutCommandDataWithBlittableJson : PutCommandDataBase<BlittableJsonReaderObject>
     {
-        public PutCommandDataWithBlittableJson(string id, string changeVector, BlittableJsonReaderObject document)
-            : base(id, changeVector, document)
+        public PutCommandDataWithBlittableJson(string id, string changeVector, BlittableJsonReaderObject document, ForceRevisionStrategy forceRevisionCreationStrategy = ForceRevisionStrategy.None)
```

Comment:
rename the argument to 'revisionCreationStrategy'

Reply:
or maybe rename the param to just 'strategy' ?